src/screens/AudioScreen.tsx

```diff
@@ -26,94 +27,96 @@ const AudioScreen = ({ navigation }): JSX.Element => {
   }
 
   return (
-    <View style={styles.background}>
-      <View style={styles.navContainer}>
-        <TouchableOpacity onPress={() => navigation.navigate('Home')}>
-          <Image
-            source={require('../assets/backArrow.png')}
-            style={styles.buttonStyle}
-          />
-        </TouchableOpacity>
-      </View>
-      <View style={styles.audioDetailContainer}>
-        <Image source={{uri: playerImageURL}} style={styles.playerImageStyle} />
-        <Text style={styles.audioTitleStyle}>{playerText}</Text>
-      </View>
-      <View style={styles.audioPlayerContainer}>
-        <Slider
-          style={styles.sliderStyle}
-          minimumValue={0}
-          maximumValue={audioLength}
-          minimumTrackTintColor="white"
-          thumbTintColor="white"
-          value={sliderValue}
-          onValueChange={handleSliderChange}
-        />
-        <Text style={styles.textStyle}>
-          {((sliderValue / audioLength) * 100).toFixed(0)}%
-        </Text>
-        <View style={styles.audioControllerContainer}>
-          <TouchableOpacity onPress={() => {}}>
-            <Image
-              source={require('../assets/backtrack.png')}
-              style={styles.buttonStyle}
-            />
-          </TouchableOpacity>
-          <TouchableOpacity
-            onPress={() => {
-              if (sliderValue >= 5) {
-                setSliderValue(sliderValue - 5)
-              } else {
-                setSliderValue(0)
-              }
-            }}>
+    <GestureHandlerRootView style={{ flex: 1 }}>
```

Comment:
the only change is the addition of <GestureHandlerRootView> wrapper around the original UI